models/core/transfers.sql

```diff
@@ -0,0 +1,49 @@
+{{ 
+    config(
+        materialized='incremental', 
+        unique_key= "CONCAT_WS('-', 'log_id', 'block_id', 'tx_hash', coalesce('from_address',''), coalesce('to_address',''), coalesce('contract_address',''), coalesce('raw_amount',-1))",
+        incremental_strategy = 'delete+insert',
+        tags=['core'],
+        cluster_by = ['block_timestamp']
+    ) 
+}}
+
+with 
+
+logs as (
```

Comment:
bring to the same line with `with`

Reply:
I copied this from the logs table. I think you will need to change this too @forgxyz :)

Reply:
Resolved with commit: https://github.com/MetricsDAO/harmony_dbt/pull/19/commits/ff697e761495024a501d7b4a077ab8dcb0b215c2